packages/requests/src/lib/requests-result.ts

```diff
@@ -272,6 +273,13 @@ export function trackRequestResult<TData>(
               }
 
               updateRequestResult(key, newResult);
+
+              if (additionalKeysFn) {
+                const mainKey = resolveKey(key);
+                for (const keys of additionalKeysFn(sourceData)) {
```

Comment:
Why you are not using the updateRequestResult function?

Reply:
I did at first, but realized that it does nothing if nothing is cached for the key, and I simply want to cache the same request result for another key.